ledger/accountdb.go

```diff
@@ -3633,11 +3637,14 @@ func accountsNewRoundImpl(
 		}
 	}
 
+	updatedBoxes = make(map[string]persistedBoxData)
```

Comment:
@algoidurovic Does the following _suggestion_ seem plausible to you?

I imagine (have not researched closely) it _hardly_ impacts performance, though it seems we can easily + accurately pre-allocate `updatedBoxes` here.

```suggestion
	updatedBoxes = make(map[string]persistedBoxData, len(kvPairs))
```

Reply:
Seems reasonable, I'll add it.